### ISSUE TITLE
Excel 2003: Arrow keys can't be used to read charts

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#5242]
**Reported by jkinnunen on 2015-07-24 05:18**
In Excel 2003, when a chart is chosen from the Elements list, NVDA starts incorrectly reading the chart by itself, and arrow keys can't be used at all to read it.

**Comment by bhavyashah** [comment]
To summarize:
* Arrowing and other support for Excel charts will not be supported for Excel 2003 (probably due to 2003 being too old a version to prioritize and specifically work on).
* The lack of support of NVDA for charts in Excel 2003 requires to be documented.
* Listing of charts (probably in the Elements List dialog?) should be disabled for Excel 2003 since we do not support them there anyways.

**Comment by leonardder** [comment]
>     * The lack of support of NVDA for charts in Excel 2003 requires to be documented.

I assume that most people will safely assume that support for chards in Excel 2003 might be lacking because of its age. We closed several issues about Office 2003, Outlook Express, Windows Mail, etc. in the past, and I don't think that this is an exception. Therefore, closing this for now.
